lib/stripe/core_resources/payment_intent.ex

```diff
@@ -0,0 +1,289 @@
+defmodule Stripe.PaymentIntent do
+  @moduledoc """
+  Work with [Stripe `payment_intent` objects](https://stripe.com/docs/api/payment_intents).
```

Comment:
"Allows compatibility with..."

Reply:
https://stripe.com/docs/quickstart
`Starting in September 2019, a new regulation called Strong Customer Authentication (SCA) will require businesses in Europe to request additional authentication for online payments. We recommend that all European businesses start building their Stripe integrations with either our PaymentIntents API or our new Checkout to be ready for these rule changes.`